src/App.jsx

```diff
@@ -36,6 +38,7 @@ function App() {
             element={<ArticlesList allArticles={allArticles} />}
           />
           <Route path="/articles/:article_id" element={<FullArticle />} />
+          <Route path="/articles/:article_id/comments" element={<CommentsList />} />
```

Comment:
Your plan seems to show CommentList as being a child of FullArticle rather than app. I think it makes sense to display the comments under an article rather than navigate away from it to see them. This is something you should consider changing.